mathics/builtin/structure.py

```diff
@@ -1276,5 +1281,10 @@ class ByteCount(Builtin):
 
     def apply(self, expression, evaluation):
         'ByteCount[expression_]'
-
-        return Integer(count_bytes(expression))
+        if bytecount_support == False:
```

Comment:
As of january 2016, Jython [has](https://mail.python.org/pipermail/jython-checkins/2016-January/001669.html) getsizeof() support too. I think it's just PyPy.